.travis/test.sh

```diff
@@ -14,6 +14,7 @@ echo 'Raw-SHA512-free-opencl = Y' >> john-local.conf
 echo 'XSHA512-free-opencl = Y' >> john-local.conf
 echo 'gpg-opencl = Y' >> john-local.conf
 echo 'KeePass-opencl = Y' >> john-local.conf
+echo 'tezos-opencl = Y' >> john-local.conf  # fails on CPU device under macOS 10.13.5
```

Comment:
This should be a last resort, we should try to understand why/what is happening.

Reply:
Yep, I am still debugging it (but without much luck).